.ci/common-ci-settings-and-functions.sh

```diff
@@ -53,6 +56,18 @@ export JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE=${JVM_BUILD_SERVICE_REQPROCESSOR_IMA
 export JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE_REPO=${JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE:-"quay.io/redhat-appstudio/hacbs-jvm-build-request-processor"}
 export JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE_TAG=${JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE_TAG:-"redhat-appstudio-jvm-build-service-reqprocessor-image"}
 
+export JVM_BUILD_SERVICE_JDK8_BUILDER_IMAGE=${JVM_BUILD_SERVICE_JDK8_BUILDER_IMAGE%@*}
```

Comment:
@gabemontero where are these images actually built? I will need to build the builder images now as well but I can't seem to find where this build+env var setting happens.

Reply:
@stuartwdouglas [here's the openshift-ci job config file](https://github.com/openshift/release/blob/master/ci-operator/config/redhat-appstudio/jvm-build-service/redhat-appstudio-jvm-build-service-main.yaml) where they need to be added

you can find more info in the [readme](https://github.com/redhat-appstudio/jvm-build-service/blob/main/.ci/README.md)

Reply:
@stuartwdouglas this is what we opened https://issues.redhat.com/browse/PLNSRVCE-517 after our exchange start started with my comment https://github.com/redhat-appstudio/jvm-build-service/pull/106#pullrequestreview-1042028142
and ended with your comment https://github.com/redhat-appstudio/jvm-build-service/pull/106#issuecomment-1192064239

Reply:
I tried the capture all the relevant steps in my update to the description in but of course we can talk, divide and conquer as needed if you are ready to take on https://issues.redhat.com/browse/PLNSRVCE-517

@Michkov FYI

Reply:
https://github.com/openshift/release/pull/30919

Reply:
update for anyone coming into this thread mid stream ... https://github.com/openshift/release/pull/30919 has merged

after these changes here, next step is to add the requisite updates in infra-deps (but correct me if I'm wrong @stuartwdouglas ) to consume them.

Reply:
https://github.com/redhat-appstudio/infra-deployments/pull/537 configures the config map to reference them in infra deployments.

Reply:
yep thanks for that more specif ref @stuartwdouglas 

marking resolved